sample/app/build.gradle

```diff
@@ -28,6 +28,9 @@ buildProperties {
     env {
         from System.getenv()
     }
+    cli {
+        from project.properties
```

Comment:
* **System properties** == [`System.getProperties()`](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getProperties()): Determines the current system properties. A property can be provided at command line via `-Dkey=value`, or at runtime using [`System.setProperty(String, String)`](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#setProperty(java.lang.String,%20java.lang.String))
* **System environment properties** (or environment variables) == [`System.getenv()`](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getenv()): Returns an unmodifiable string map view of the current system environment. The environment is a system-dependent mapping from names to values which is passed from parent to child processes. Usually you would set a variable via `export FOO=bar`.